lib/rucio/core/replica.py

```diff
@@ -1128,9 +1191,8 @@ def _resolve_collection_files(temp_table, *, session: "Session"):
             ['scope', 'name'],
             rucio.core.did.list_child_dids_stmt(temp_table, did_type=DIDType.FILE)
```

Comment:
This operation should be done outside the statement I think, this statement is rough on the eyes.

Reply:
Changed to:

```
        selectable = rucio.core.did.list_child_dids_stmt(temp_table, did_type=DIDType.FILE)

        stmt = insert(
            resolved_files_temp_table
        ).from_select(
            ['scope', 'name'],
            selectable
        )
```